src/views/DrefOperationalUpdateExport/index.tsx

```diff
@@ -49,6 +55,58 @@ const colorMap: Record<DisasterCategory, string> = {
     [DISASTER_CATEGORY_RED]: styles.red,
 };
 
+const plannedInterventionSortedList = [
+    'shelter_housing_and_settlements',
+    'livelihoods_and_basic_needs',
+    'multi-purpose_cash',
+    'health',
+    'water_sanitation_and_hygiene',
+    'protection_gender_and_inclusion',
+    'education',
+    'migration',
+    'risk_reduction_climate_adaptation_and_recovery_',
+    'community_engagement_and_accountability',
+    'environmental_sustainability',
+    'coordination_and_partnerships',
+    'secretariat_services',
+    'national_society_strengthening',
+] satisfies (PlannedIntervention['title'])[];
+
+const identifiedNeedsAndGapsSortedList = [
+    'shelter_housing_and_settlements',
+    'livelihoods_and_basic_needs',
+    'multi_purpose_cash_grants',
+    'health',
+    'water_sanitation_and_hygiene',
+    'protection_gender_and_inclusion',
+    'education',
+    'migration',
+    'risk_reduction_climate_adaptation_and_recovery',
+    'community_engagement_and _accountability',
+    'environment_sustainability ',
+] satisfies (IdentifiedNeedsAndGaps['title'])[];
+
+const nsActionsSortedList = [
+    'shelter_housing_and_settlements',
+    'livelihoods_and_basic_needs',
+    'multi-purpose_cash',
+    'health',
+    'water_sanitation_and_hygiene',
+    'protection_gender_and_inclusion',
+    'education',
+    'migration',
+    'risk_reduction_climate_adaptation_and_recovery',
+    'community_engagement_and _accountability',
+    'environment_sustainability ',
+    'coordination',
+    'national_society_readiness',
+    'assessment',
+    'resource_mobilization',
+    'activation_of_contingency_plans',
+    'national_society_eoc',
+    'other',
+] satisfies (NsActions['title'])[];
+
```

Comment:
Looking at it now, we could in fact create a map of the keys as 
```{
rankedKeys: {
    shelter_housing_and_settlements: 1,
    livelihoods_and_basic_needs: 2,
}
}
```
and instead of using `indexOf` we could have done `rankedKeys[a.title] - rankedKeys[b.title]` in the sort function below.
This could be a bit performant. :smile: